src/routes/connect/+page.svelte

```diff
@@ -273,31 +247,17 @@
       }
     }
 
-    & .features-banner {
```

Comment:
and this?

Reply:
design changes should probably be separate from fixes

Reply:
It was ultimately decided to completely remove the features banner, which included the nav icon, instead of just removing the nav icon individually. This meant the styles were no longer used.

I agree the fix should have been separated, but it was something necessary for the nav icon removal originally. I suppose I should have reverted and done it separately, since the fix wasn't necessary after removing the banner.